srcs/incs/WebServer.hpp

```diff
@@ -31,11 +35,13 @@ class WebServer {
 		bool	PopulateServers_();
 		void	AddListeningSocketsToMasterSet_();
 		void	SetMaxSocket_(int curr_sd);
-		std::vector<Server>::iterator	FindListeningServer(int sd);
-		void	AcceptNewConnection_(std::vector<Server>::iterator server_it);
-		std::vector<Server>::iterator	FindConnectionServer(int sd);
+		void	AcceptNewConnection_(int sd);
```

Comment:
The opposite here, since these methods are public, shouldn’t we remove underscore?

Reply:
Good catch! I've added underscores to these methods (they are private and appear to be public because of Github's diff folding)